python/tests/conftest.py

```diff
@@ -38,6 +38,7 @@ def spark(tmp_path_factory) -> SparkSession:
             "spark.sql.extensions",
             "ai.eto.rikai.sql.spark.RikaiSparkSessionExtensions",
         )
+        .config("rikai.sql.ml.registry.uri.root", "mlflow:/")
```

Comment:
are we going to make `mlflow` the main test case?

Reply:
what do you think would be the best way to make this limited in scope? this config variable is used once in Registry.registerAll and then that's it.

Reply:
can we provide a `conftest.py`  for `rikai/python/tests/mlflow`?

Reply:
was hoping to avoid having multiple Spark sessions in the same test suite. but maybe unavoidable.
i guess the only alternative is adding some reset SQL command to clear the registryMap

Reply:
oh, what i suggested that use a separate spark session that is dedicated for `mlflow`